tests/DocumentationExamplesTest.php

```diff
@@ -0,0 +1,156 @@
+     */
+    public function testOneToMany(): void
+    {
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/one-to-many/Planet.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/one-to-many/Moon.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/RelationshipController.php';
+
+        // Clear the database
+        Planet::truncate();
+
+        $controller = new RelationshipController();
+        $controller->oneToMany();
+
+        $planet = Planet::first();
+        $this->assertInstanceOf(Planet::class, $planet);
+        $this->assertCount(2, $planet->moons);
+        $this->assertInstanceOf(Moon::class, $planet->moons->first());
+
+        $controller->planetMoonsDynamic();
+    }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testManyToMany(): void
+    {
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/many-to-many/Planet.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/many-to-many/SpaceExplorer.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/RelationshipController.php';
+
+        // Clear the database
+        Planet::truncate();
+        SpaceExplorer::truncate();
+
+        $controller = new RelationshipController();
+        $controller->manyToMany();
+
+        $planet = Planet::where('name', 'Earth')->first();
+        $this->assertInstanceOf(Planet::class, $planet);
+        $this->assertCount(3, $planet->visitors);
+        $this->assertInstanceOf(SpaceExplorer::class, $planet->visitors->first());
+
+        $explorer = SpaceExplorer::where('name', 'Jean-Luc Picard')->first();
+        $this->assertInstanceOf(SpaceExplorer::class, $explorer);
+        $this->assertCount(3, $explorer->planetsVisited);
+        $this->assertInstanceOf(Planet::class, $explorer->planetsVisited->first());
+
+        $controller->manyToManyDynamic();
+    }
+
+    /**
+     * @runInSeparateProcess
+     * @preserveGlobalState disabled
+     */
+    public function testEmbedsMany(): void
+    {
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/embeds/Cargo.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/embeds/SpaceShip.php';
+        require_once __DIR__ . '/../docs/includes/eloquent-models/relationships/RelationshipController.php';
+
+        // Clear the database
+        SpaceShip::truncate();
+
+        $controller = new RelationshipController();
+        $controller->embedsMany();
+
+        $spaceship = SpaceShip::first();
+        $this->assertInstanceOf(SpaceShip::class, $spaceship);
+        $this->assertCount(2, $spaceship->cargo);
+        $this->assertInstanceOf(Cargo::class, $spaceship->cargo->first());
+    }
+
+    /**
+     * @r unInSeparateProcess
+     * @ preserveGlobalState disabled
```

Comment:
Are the spaces intentional? We should either use the correct annotations or remove the docblock.

```suggestion
     * @runInSeparateProcess
     * @preserveGlobalState disabled
```

Reply:
Good catch. Fixed.